www-apps/sonarr/files/sonarr.init

```diff
@@ -0,0 +1,48 @@
+#!/sbin/openrc-run
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+depend() {
+        need net
+}
+
+run_dir="/run/sonarr"
+pidfile="$run_dir/sonarr.pid"
+etc_dir="/etc/sonarr"
+var_dir="/var/lib/sonarr"
+log_dir="/var/log/sonarr"
+log_file="sonarr.log"
+program="$APP_DIR/Sonarr.exe"
+bin=`which mono`
```

Comment:
If need to do this, add which to RDEPEND, but ideally use command -v instead.

Reply:
done, thank you for pointing that out!